magento/inventory
# 36555: Magento Community Edition: Version 2.4.5 - Import Stock Source…

Related to this issue: [36555: Magento Community Edition: Version 2.4.5 - Import Stock Sources Added Extra Column in Import CSV File, Validation message is not understandable to the end users [magento/magento2 repository]](https://github.com/magento/magento2/issues/36555)





### Description (*)


### Fixed Issues (if relevant)

1. Fixes magento/magento2 #36555:  Magento Community Edition: Version 2.4.5 - Import Stock Sources Added Extra Column in Import CSV File, Validation message is not understandable to the end users

### Manual testing scenarios (*)

1. Login into admin > System > Data Transfer > Import, Select "Stock Sources" option from Entity Type drop down.
2. Select "Add/Update Complex Data" option from Import Behaviour drop down.
3. Rest of fields leave as it is.
4. Upload prepared CSV file for import the Stock Sources as per the below attachment. Or by adding new/extra column with name of "Test" (Column name should start with capital letter)
5. Click on check data, "columnNameInvalid" Validation message is not understandable to the end users. While same case checked with "Advanced Pricing" entity, Validation message is easily understandable to the end users.
![image](https://user-images.githubusercontent.com/16774074/207401969-7d903505-844e-47eb-b75c-33f07b4176bc.png)
![image](https://user-images.githubusercontent.com/16774074/207402043-3ee387f4-1262-4226-9070-0034135e32e8.png)

### Questions or comments


### Contribution checklist (*)
